plugin-dev/Source/Sentry/Private/Desktop/SentrySubsystemDesktop.cpp

```diff
@@ -104,11 +95,24 @@ void SentrySubsystemDesktop::InitWithSettings(const USentrySettings* settings, U
 	}
 
 #if PLATFORM_WINDOWS
-	sentry_options_set_handler_pathw(options, *FPaths::ConvertRelativePathToFull(HandlerPath));
-	sentry_options_set_database_pathw(options, *FPaths::ConvertRelativePathToFull(DatabasePath));
+	if(!FSentryModule::Get().IsMarketplaceVersion())
+	{
+		if(!FPaths::FileExists(GetHandlerPath()))
+		{
+			UE_LOG(LogSentrySdk, Log, TEXT("Crashpad executable couldn't be found so Breakpad will be used instead. "
```

Comment:
I don't get how we fall back to `Breakpad`. Is that an implicit fallback during initialization due to a missing `Crashpad` executable? Because the non-existing path still gets set.